Week3/js-exercises-week3/removeDuplicates.js

```diff
@@ -0,0 +1,18 @@
+const letters = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c', 'b'];
+
+function takeOutTheDuplicates(arr) {
+    let cleanArr = [];
+    for (let letter of arr) {
+
+        let existence = cleanArr.includes(letter)
+        if (existence == false) {
```

Comment:
No need to declare a variable here 
```suggestion
        if (!cleanArr.includes(letter)) {
```